config/database.yml

```diff
@@ -1,5 +0,0 @@
-# This config is used when setting up a basic rails environment for running the
```

Comment:
Did you mean to include this change in this PR? I'm not opposed to it as long as the tests pass, but it also looks like the sole change in https://github.com/newrelic/newrelic-ruby-agent/pull/2545

Reply:
Ah, I didn't switch back to dev before creating the branch. Fixed by merging that other branch to dev and then dev to here. Thanks for catching it!